packages/create-modular-react-app/src/cli.ts

```diff
@@ -5,16 +5,23 @@ import { program } from 'commander';
 import * as fs from 'fs-extra';
 import { JSONSchemaForNPMPackageJsonFiles as PackageJson } from '@schemastore/package';
 
+// this is a bit gross - but there's no better way of doing this.
+let verbose = false;
```

Comment:
Could we read it from `program.argv` instead? We allow it as an option so that commander doesn't yell at us and we can pass the option to `createModularApp` in the action but outside of it, we can read `--verbose` from `process.argv`.

Reply:
Yeah - seems legit to me. As you said `commander` will validate any args immediately anyway.